src/pykeen/models/meta/filtered.py

```diff
@@ -69,6 +68,7 @@ def __init__(
         """
         # avoid cyclic imports
         from .. import model_resolver
+        from ...evaluation.evaluator import prepare_filter_triples
```

Comment:
this is a utility function that doesn't rely on anything in the evaluation module so maybe just move to `pykeen.utils`?